### PR TITLE
Added specs for kernel configurations

### PR DESCRIPTION
**Pull request by vishwanathjadhav**

### CODE REVIEW

**Comment by patchkez** [comment]
test this please

**Comment by vishwanathjadhav** [comment]
@patchkez - Done

File: uploader.json [hunk 1]
```diff
@@ -3335,6 +3335,11 @@
             "symbolic_name": "amq_broker",
             "pattern": []
         },
+        {
+            "glob": "/boot/config-*",
+            "symbolic_name": "kernel_config",
+            "pattern": []
+        },
```
Comment (bfahr) [inline uploader.json]: Glob files won't work for the V1 client in `uploader.json`, you need to add a `file` spec instead in the form `"/boot/()config-.*"`.

**Comment by vishwanathjadhav** [comment]
Covered in https://github.com/RedHatInsights/insights-core-assets/pull/290

File: uploader.json (hunk 1 above)
Comment (vishwanathjadhav) [reply uploader.json]: Done

**Pull request closed without merging**